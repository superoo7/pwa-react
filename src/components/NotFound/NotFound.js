import React from 'react'

const NotFound = () => (
  <div>
    <h2>Error 404</h2>
    <h5>The page is not found</h5>
  </div>
)

export default NotFound
