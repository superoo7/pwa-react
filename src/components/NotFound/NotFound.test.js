import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './NotFound'
import toJson from 'enzyme-to-json'


test('NotFound component should render as expected', () => {
  const component = shallow(<NotFound />)
  const tree = toJson(component)
  // console.log(tree)
  expect(tree).toMatchSnapshot()
})
