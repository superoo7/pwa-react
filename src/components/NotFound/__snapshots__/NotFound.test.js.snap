// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render as expected 1`] = `
<div>
  <h2>
    Error 404
  </h2>
  <h5>
    The page is not found
  </h5>
</div>
`;
